img/index.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <a href="#home">Eyasu</a>
+
+
+        <nav>
+            <img class="nav-pic" src="img/union.png" alt="">
+
+
+        </nav>
+    </header>
+    <section id="home" class="home">
+        <div class="home-content">
+            <h1>
+                I'm Eyasu <br> Glad to see <br> you!
+            </h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+
+
+            <a href="twitter.com"><img src="img/twitter.png" alt=""></a>
+            <a href="linkedin.com"><img src="img/Linkedin_icon.png" alt=""></a>
+            <a href="medium.com"><img src="img/medium.png" alt=""></a>
+            <a href="github.com"><img src="img/github.png" alt=""></a>
+            <a href="angellist.com"><img src="img/angellist.png" alt=""></a>
+
+        </div>
+
+    </section>
+    <section class="work">
+        <div class="grid-item grid-item1">
+            <div class="work-img1">
+
+            </div>
+            <h2>Tonic</h2>
+            <div class="work-content work-content1">
+
+                <p>CANCOPY</p>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="work-description">
+                <p class="p-1">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="categories">
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+            </div>
+            <div class="button">
+                <p>
+                    See Project
+                </p>

```

Comment:
- [ ] please kindly use a `button tag` here and add a link place holder to it. make the hover and style to be like the figma version